google-cloud-compute-v1/acceptance/google/cloud/compute/v1/query_params_smoke_test.rb

```diff
@@ -0,0 +1,113 @@
+require "minitest/autorun"
+
+require "google/cloud/compute/v1/instance_group_managers"
+require "google/cloud/compute/v1/instance_templates"
+
+require "google/cloud/compute/v1/zone_operations"
+require "google/cloud/compute/v1/global_operations"
+
+class QueryParamsSmokeTest < Minitest::Test
+  def setup
+    @default_zone = "us-central1-a"
+    @default_project = ENV["COMPUTE_TEST_PROJECT"]
+
+    @templates_client = ::Google::Cloud::Compute::V1::InstanceTemplates::Rest::Client.new
+    @igm_client = ::Google::Cloud::Compute::V1::InstanceGroupManagers::Rest::Client.new
+
+    @zonal_ops_client = ::Google::Cloud::Compute::V1::ZoneOperations::Rest::Client.new
+    @global_ops_client = ::Google::Cloud::Compute::V1::GlobalOperations::Rest::Client.new
+
+    @image_name =  "projects/debian-cloud/global/images/family/debian-10"
+    skip "COMPUTE_TEST_PROJECT must be set before running this test" if @default_project.nil?
+  end
+
+  # Smoke-test the following behaviours: 
+  #   1) set body field to nil
+  #   2) set query param to nil
+  def test_query_params
+    template_name = "rbgapic#{rand 10_000}"
+    igm_name = "rbgapic#{rand 10_000}"
+    template_resource = {
+      name: template_name,
+      properties: {
+        disks: [
+          {
+            initialize_params:
+              {
+                source_image: @image_name
+              },
+            boot: true,
+            auto_delete: true,
+            type: "PERSISTENT"
+          }
+        ],
+      machine_type: "n1-standard-1",
+      network_interfaces: [{ access_configs: [{ name: "default", type: "ONE_TO_ONE_NAT" }] }]
+      }
+    }
+
+    begin
+      puts "inserting instance template #{template_name}"
+      gapic_op = @templates_client.insert project: @default_project, instance_template_resource: template_resource
+      wait_for_global_op gapic_op, "insert instance template #{template_name}"
+
+      igm_resource = {
+        base_instance_name: "rbgapicinst",
+        target_size: 0,
+        instance_template: gapic_op.operation.target_link,
+        name: igm_name
+      }
+
+      puts "inserting instance_group_manager #{igm_resource[:name]}"
+      op = @igm_client.insert project: @default_project, zone: @default_zone, instance_group_manager_resource: igm_resource
+      wait_for_zonal_op op, "insert instance_group_manager #{igm_resource[:name]}"
+
+      igm = @igm_client.get project: @default_project, zone: @default_zone, instance_group_manager: igm_name
+      assert_equal igm.target_size, 0
+
+      resize_op = @igm_client.resize project: @default_project, zone: @default_zone, instance_group_manager: igm_name,
+                                    size: 1
```

Comment:
nit: indentation. Also an orphan. I'd move the previous argument onto this line as well. Or if Rubocop isn't running on these files anyway, then just move everything onto the same line.

Reply:
:+1: